RecoTracker/LSTCore/interface/Constants.h

```diff
@@ -32,6 +32,9 @@ namespace lst {
                                                    alpaka_common::Vec1D(static_cast<alpaka_common::Idx>(nElements)));
   }
 
+  // Named constants for pixelTypes
+  enum kPixelType { highPt = 0, lowPtPosCurv = 1, lowPtNegCurv = 2 };
```

Comment:
```suggestion
  enum PixelType { kHighPt = 0, kLowPtPosCurv = 1, kLowPtNegCurv = 2 };
```
type names should be capitalized while the constant values should be prefixed with `k`

Reply:
I'm guessing the switch from `int8_t` to `int` was triggered by an enum being implicitly int.
```suggestion
  enum PixelType : int8_t { kHighPt = 0, kLowPtPosCurv = 1, kLowPtNegCurv = 2 };
```

Reply:
> I'm guessing the switch from int8_t to int was triggered by an enum being implicitly int.

Indeed. I will try this approach, thanks!